bin/migrate-oats-data/noi/sql/notice_of_intent_submission/statuses/submitted_to_alc_incomplete/validation_script.sql

```diff
@@ -7,12 +7,10 @@ WHERE status_type_code = 'SUIN'
     AND effective_date IS NOT NULL;
 -- total count of expected 'AKI' - Submitted to ALC Incomplete status
 WITH noi_components_grouped AS (
-    SELECT oaac.alr_application_id
-    FROM oats.oats_alr_appl_components oaac
-        JOIN oats.oats_alr_applications oaa ON oaa.alr_application_id = oaac.alr_application_id
+    SELECT oaa.alr_application_id
+    FROM oats.alcs_etl_applications_nois oaa
```

Comment:
Acknowledging that oaa is not the best name but allows minimal code change/disruption.
I believe it is fine to keep as is (same for previous files)